stock_lock_lot/models/stock_quant.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+from openerp import models, fields
+
+
+class StockQuant(models.Model):
+    _inherit = 'stock.quant'
+
+    locked = fields.Boolean(
+        string='Blocked', related="lot_id.locked", default=False,
+        store=True)
+
+    def quants_get(self, cr, uid, location, product, qty, domain=None,
+                   restrict_lot_id=False, restrict_partner_id=False,
+                   context=None):
```

Comment:
I suppose that was to ensure compatibility with the v7-API method in the standard.
Can we write it another way?